src/MeetingsSDKAdapter.js

```diff
@@ -0,0 +1,102 @@
+import {MeetingsAdapter} from '@webex/component-adapter-interfaces';
+import {Observable} from 'rxjs';
+import {publish, refCount} from 'rxjs/operators';
+
+export const ROOM_UPDATED_EVENT = 'updated';
+
+/**
+ * The `MeetingsSDKAdapter` is an implementation of the `MeetingsAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to create and join webex meetings.
+ *
+ * @export
+ * @class MeetingsSDKAdapter
+ * @extends {MeetingsAdapter}
+ */
+export default class MeetingsSDKAdapter extends MeetingsAdapter {
+  constructor(datasource) {
+    super(datasource);
+    this.meetingsObservables = {};
+    this.meetingControls = {};
+  }
+
+  connect() {
+    // Check if the meetings is registred
+    if (!this.datasource.meeting.registered) {
+      this.datasource.meetings.register();
+    }
+
+    this.registerEvents();
+    this.syncMeetings();
+  }
+
+  syncMeetings() {
+    this.datasource.meetings.syncMeetings();
```

Comment:
We should probably await this since it returns a promise:
```suggestion
    await this.datasource.meetings.syncMeetings();
```